optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/migrate313to86/DeleteEventIndicesAndTemplates86IT.java.out

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+// TODO recreate C8 IT equivalent of this with #13337
+//package io.camunda.optimize.upgrade.migrate313to86;
+//
+//import static org.assertj.core.api.Assertions.assertThat;
+//import java.util.List;
+//import lombok.SneakyThrows;
+//import org.junit.jupiter.api.Test;
+//
+//public class DeleteEventIndicesAndTemplates86IT extends AbstractUpgrade86IT {
```

Comment:
is the plan to reenable this?

Reply:
Depends on how fast we are with the C8 IT migration, I added them so that I can easily the migration logic locally and am planning to rerun them whenever I add more migration steps just to double check nothing is changing. Can just keep them local/uncommitted as well though

Reply:
I can comment them back in if we want to keep them as is